src/sentry/data/samples/android.json

```diff
@@ -24,6 +24,7 @@
       "screen_density": 3,
       "type": "device",
       "charging": true,
+      "language": "en_US",
```

Comment:
I think we are mixing here language and locale. Language is only about the language and would be `en`. Locale can also contain the country, but the language has no info about the country.

> Locale involves cultural adaptation and tailoring your content for specific audiences in other countries. It means going beyond literal translations that might sound funny to native speakers of that region; instead, you need to get acquainted with their customs and habits to convey meaning without sounding too strange appropriately.
Taken from https://simplelocalize.io/blog/posts/language-vs-locale/

Reply:
```suggestion
      "language": "en",
```